### PR TITLE
tweak(platform): Update Nav Menu styling 

### PR DESCRIPTION
**Pull request by Swiftyos**
### Background

Some simple nav menu styling changes

### Changes 🏗️

Screenshot 2024-09-24 at 13 05 55
Screenshot 2024-09-24 at 13 06 09

Only for the new autogpt platform, currently in autogpt_platform/

- Create from scratch and execute an agent with at least 3 blocks
- Import an agent from file upload, and confirm it executes correctly
- Upload agent to marketplace
- Import an agent from marketplace and confirm it executes correctly
- Edit an agent from monitor, and confirm it executes correctly

### CODE REVIEW

**Comment by Swiftyos** [comment]
@Significant-Gravitas/maintainers Given we are so close to release. Please can you be extra diligent during reviews to ensure no bugs slip through

**Comment by majdyz** [comment]
image

This is what it looks like on my end.
Browser: chromium arc

**Comment by majdyz** [comment]
Seems like the my lock file was poisoned due to the yarn.lock update, it's working now 👍 

image

Is the rounding and the drop shadow necessary ? it makes the navbar look detached.
But I have no issues code-wise.

File: autogpt_platform/frontend/src/components/NavBar.tsx [hunk 1]
```diff
@@ -40,64 +43,58 @@ export async function NavBar() {
...
         </Sheet>
-        <nav className="hidden md:flex md:flex-row md:items-center md:gap-5 lg:gap-6">
+        <nav className="hidden md:flex md:flex-row md:items-center md:gap-7 lg:gap-8">
+          <div className="flex h-10 w-20 flex-1 flex-row items-center justify-center gap-2">
+            <a href="https://agpt.co/">
+              <Image
+                src="/AUTOgpt_Logo_dark.png"
+                alt="AutoGPT Logo"
+                width={100}
+                height={40}
```
Comment (majdyz) [inline autogpt_platform/frontend/src/components/NavBar.tsx:73]: Changing height from 20 to 40, won't affect anything since it's maintaining its ratio.

**Pull request merged**